### PR TITLE
[Dist/Tizen] Enable nnstreamer-ncsdk2

### LINKED ISSUE #1845: [Tizen] Enable nnstreamer-ncsdk in Tizen platform
This is a sub-item of https://github.com/nnsuite/nnstreamer/issues/1709.

As libmvnc (ncsdk2) has been published via https://build.tizen.org/package/show/devel:AIC:Tizen:5.0:nnsuite/libmvnc, it is possible to enable nnstreamer-ncsdk in Tizen platform.

### PR DESCRIPTION
**Pull request by wooksong**
Resolves #1845 
See also #1709

This PR addresses #1845, which is about enabling the tensor filter extension for Intel Movidius Neural Compute stick SDK2 (NCSDK2) in Tizen.

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Note that you need to take https://github.com/nnsuite/tizenport-ncsdk into Tizen:Unified before integrating this into Tizen.

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -76,6 +76,7 @@ BuildRequires:	pkgconfig(capi-privacy-privilege-manager)
 BuildRequires:	pkgconfig(capi-system-info)
 BuildRequires:	pkgconfig(capi-base-common)
 BuildRequires:	pkgconfig(dlog)
+BuildRequires:	pkgconfig(libmvnc)
```
Comment (myungjoo) [inline packaging/nnstreamer.spec]: Although this is compatible with devel:Tizen, it's not compatible with Tizen;Unified. We need to merge mvnc libraries into Tizen;Unified first. NNStreamer's master branch should be compatible with Tizen:Unified.

Comment (leemgs) [reply packaging/nnstreamer.spec]: Right, We need to establish the Movidius Neural Computing (e.g., libmvnc)  git repository in Tizen:Unified  without devel:Tizen.

Comment (myungjoo) [reply packaging/nnstreamer.spec]: How do you want the repo formed?

/adaptation/npu/mvnc?
/adaptation/npu/libmvnc?
/adaptation/npu/movidius?
???

Comment (wooksong) [reply packaging/nnstreamer.spec]: @myungjoo How about /adaptation/npu/libmvnc?

Comment (myungjoo) [reply packaging/nnstreamer.spec]: and.. looking at platform/adaptation/* (probably they've changed the rule from adaptation/* --> platform/adaptation/* recently), we may need an additional naming rule:

platform/adaptation/npu/intel/libmvnc
platform/adaptation/npu/intel/openvino
platform/adaptation/npu/google/edgeTPU
...

or

platform/adaptation/npu/intel_libmvnc
platform/adaptation/npu/intel_openvino
platform/adaptation/npu/google_edgeTPU
...

Comment (wooksong) [reply packaging/nnstreamer.spec]: IMO, intel_* or google* looks pretty as well as samsung_exynos. About the name, npu, does it look a little bit specific? How about using the ambitious word, ml? 

```
platform/adaptation/ml/intel_libmvnc
```

Comment (myungjoo) [reply packaging/nnstreamer.spec]: ml smells too much of software without hardware. In other words, "npu" makes people think of a hardware piece (thus some drivers or HALs) right away while "ml" makes people think of a pure software component (e.g., tensorflow, caffe).

Comment (myungjoo) [reply packaging/nnstreamer.spec]: Let's try platform/adaptation/npu/intel_libmvnc

Comment (wooksong) [reply packaging/nnstreamer.spec]: > Let's try platform/adaptation/npu/intel_libmvnc

Got it. I will go with that name.

Comment (myungjoo) [reply packaging/nnstreamer.spec]: https://bugs.tizen.org/browse/SO-1845

Comment (wooksong) [reply packaging/nnstreamer.spec]: > https://bugs.tizen.org/browse/SO-1845

You've already done it. Thanks.

**Comment by myungjoo** [comment]
We are almost there:
https://bugs.tizen.org/browse/SO-1845
https://bugs.tizen.org/browse/SO-1850

**Comment by myungjoo** [comment]
It looks like what you want will appear at https://git.tizen.org/cgit/platform/adaptation/npu/ tomorrow. If it doesn't show up until tomorrow noon, let's nudge SCM guys.

**Comment by wooksong** [comment]
> It looks like what you want will appear at https://git.tizen.org/cgit/platform/adaptation/npu/ tomorrow. If it doesn't show up until tomorrow noon, let's nudge SCM guys.

Not yet:(
But TGIF so that we can wait until Monday morning.

**Comment by myungjoo** [comment]
Don't worry. I've nudged the manager of the correspondent.

**Comment by myungjoo** [comment]
All the required packages are included in Tizen. It's good to go.

**Pull request merged**